static/templates/stream_settings_checkbox.hbs

```diff
@@ -1,13 +1,13 @@
 {{!-- If setting is disabled on realm level, then render setting as control-label-disabled and do not set setting value. Setting status should not change on any click handler, as it is disabled at realm level. --}}
-<div id="{{prefix}}{{setting_name}}{{suffix}}" class="sub_setting_checkbox {{#unless realm_setting_disabled}}{{#if notification_setting}}sub_notification_setting {{#if is_muted}}muted-sub{{/if}}{{/if}}{{else}}control-label-disabled{{/unless}}">
```

Comment:
@pragatiagrawal31 this can be made more readable? Three nested `if`s are here.

Reply:
No clean way to do so :/